### PR TITLE
adding KIND deployment docs

### PR DESCRIPTION
**Pull request by OmerBenHayun**

### CODE REVIEW

File: dist/README.md [hunk 1]
```diff
@@ -1,7 +1,53 @@
...
+		```bash
+		# Clone ovsdb-etcd repo
+		git clone https://github.com/IBM/ovsdb-etcd
+		cd ovsdb-etcd
+		```
+	- login to your account of common image repository e.g. docker hub or quay.
+	- update the image name on the following files (replace `omerbh\etcd` and `omerbh\server` with your image names,one for the server and one for the etcd):
+		- `tagDocker`
+		- `pushDocker`
+		- `dist/ovn-kubernetes/dist/templates/ovnkube-db.etcd.yaml.j2`
```
Comment (roytman) [inline dist/README.md:32]: this file is redundant, its content is copied to /ovnkube-db.yaml.j2

File: dist/README.md [hunk 2]
```diff
@@ -1,7 +1,53 @@
...
+		- `dist/ovn-kubernetes/dist/templates/ovnkube-db.yaml.j2`
+		- `dist/deployment/ovnkube-db.yaml`
```
Comment (roytman) [inline dist/README.md:34]: why do we need this file

File: dist/README.md [hunk 3]
```diff
@@ -1,7 +1,53 @@
...
+	- clone `ovsdb-etcd` from the main repository:
+		```bash
+		# Clone ovsdb-etcd repo
+		git clone https://github.com/IBM/ovsdb-etcd
+		cd ovsdb-etcd
+		```
+	- login to your account of common image repository e.g. docker hub or quay.
+	- update the image name on the following files (replace `omerbh\etcd` and `omerbh\server` with your image names,one for the server and one for the etcd):
+		- `tagDocker`
+		- `pushDocker`
```
Comment (roytman) [inline dist/README.md:31]: why do you need 2 separate files if `pushDocker` contains all commands of `tagDocker` ?

File: dist/README.md [hunk 4]
```diff
@@ -1,7 +1,53 @@
...
+		- `dist/ovn-kubernetes/dist/templates/ovnkube-db.etcd.yaml.j2`
+		- `dist/ovn-kubernetes/dist/templates/ovnkube-db.yaml.j2`
+		- `dist/deployment/ovnkube-db.yaml`
+	- **optional step**: if you cloned the `ovn-kubernetes` repository to another different folder edit the `Makefile` and upadte `OVN_KUBERNETES_ROOT` accordingly.
```
Comment (roytman) [inline dist/README.md:35]: You have not defined `OVN_KUBERNETES_ROOT` , and of course, ovn-kubernetes is cloned to another repository.

Comment (OmerBenHayun) [reply dist/README.md]: added the relevant environment variable to the documentation.

File: dist/README.md (hunk 3 above)
Comment (OmerBenHayun) [reply dist/README.md]: added the relevant environment variable to the documentation.

File: dist/README.md (hunk 1 above)
Comment (OmerBenHayun) [reply dist/README.md]: changed this part on the docs. this part has removed.

File: dist/README.md (hunk 2 above)
Comment (OmerBenHayun) [reply dist/README.md]: this part has been removed from the docs.

**Comment by OmerBenHayun** [comment]
@roytman please note:
1. I removed the last sentence from the old `README.md` file
`The ovn-kubernetes files are from commit e567a39c76930a26bdf1f312c36fa716f18df9a7`
is that OK? 
2. I change the default value of the environment variable  `OVN_KUBERNETES_ROOT ` on the `Makefile` to be the same directory that appear on the official docs of [OVN kub](https://github.com/ovn-org/ovn-kubernetes)

File: Makefile [hunk 5]
```diff
@@ -1,7 +1,7 @@
 ROOT_DIR := .
 SERVER_EXECUTEBLE := "$(ROOT_DIR)/pkg/cmd/server/server"
 
-OVN_KUBERNETES_ROOT ?= $(ROOT_DIR)/../../ovn-org/ovn-kubernetes
+OVN_KUBERNETES_ROOT ?= $HOME/work/src/github.com/ovn-org/ovn-kubernetes
```
Comment (roytman) [inline Makefile:4]: Why did you replace it? it is wrong.

File: dist/README.md [hunk 6]
```diff
@@ -1,10 +1,54 @@
...
+	- login to your account of common image repository e.g. docker hub or quay.
+	- set the environment variable `OVSDB_ETCD_REPOSITORY` with your image repository name. for example:
+		```bash
+		export OVSDB_ETCD_REPOSITORY=omerbh
+		```
+	- set the environment variable `OVN_KUBERNETES_ROOT` to the location of OVN kubernetes repository. for example:
+		```bash
+		export OVN_KUBERNETES_ROOT=$HOME/go/src/github.com/ovn-org/ovn-kubernetes
+		```
+	    (default value : `$HOME/work/src/github.com/ovn-org/ovn-kubernetes`)
```
Comment (roytman) [inline dist/README.md:36]: please return the original default value

**Review by roytman — CHANGES_REQUESTED** [review]

File: dist/README.md [hunk 7]
```diff
@@ -1,10 +1,54 @@
...
+		```
+1. prepare ovsdb-etcd
+	- clone `ovsdb-etcd` from the main repository.
+		```bash
+		# Clone ovsdb-etcd repo
+		git clone https://github.com/IBM/ovsdb-etcd
+		cd ovsdb-etcd
+		```
+	- login to your account of common image repository e.g. docker hub or quay.
+	- set the environment variable `OVSDB_ETCD_REPOSITORY` with your image repository name. for example:
```
Comment (roytman) [inline dist/README.md:28]: set the environment variable `OVSDB_ETCD_REPOSITORY` with your **docker.io** image repository name. for example:

File: Makefile (hunk 5 above)
Comment (OmerBenHayun) [reply Makefile]: as we talked earlier I replace `OVN_KUBERNETES_ROOT `  to match the path on [OVN kubernetes](https://github.com/ovn-org/ovn-kubernetes). any way , I revert this change.

File: dist/README.md (hunk 6 above)
Comment (OmerBenHayun) [reply dist/README.md]: Done, Thanks.

File: dist/README.md (hunk 7 above)
Comment (OmerBenHayun) [reply dist/README.md]: Done, Thanks.

**Pull request merged**